ansible/roles/deploy/templates/config.json

```diff
@@ -3,10 +3,10 @@
   "job_output_dir": "/home/{{app_user}}/server/job-output",
   "path_to_db": ":memory:",
   "webhook_secret": "{{ webhook_secret }}",
-  "github_app_id": 405053,
-  "github_app_key": "{{ github_app_key }}",
-  "github_repo_owner": "rustls",
-  "github_repo_name": "rustls",
+  "github_app_id": {{ github_app_id }},
+  "github_app_key": "{{ github_app_key.replace('\n', '\\n') }}",

```

Comment:
Smart, I didn't think of this and manually removed the newlines from my private key (somehow the PEM parser didn't complain)